operators.go

```diff
@@ -62,6 +62,14 @@ func Neq(col SchemaField, value interface{}) Condition {
 	}
 }
 
+// Like returns a condition that will be true when `col` matches the given value.
+// See https://www.postgresql.org/docs/9.6/static/functions-matching.html.
+func Like(col SchemaField, value string) Condition {
+	return func(schema Schema) squirrel.Sqlizer {
+		return like{col.QualifiedName(schema), value}
```

Comment:
Use `colOp` instead of custom type https://github.com/src-d/go-kallax/blob/master/operators.go#L106